packages/app-accounts/src/Overview.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017-2019 @polkadot/app-staking authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { ComponentProps } from './types';
+import React from 'react';
+import styled from 'styled-components';
+import keyring from '@polkadot/ui-keyring';
+
+import Account from './Account';
+
+type Props = ComponentProps;
+
+const Wrapper = styled.div`
+  .accounts {
+    display: flex;
+    flex-wrap: wrap;
+
+    .spacer {
+      flex: 1 1;
+      margin: .25rem;
+      padding: 1rem 1.5rem;
+    }
+  }
+`;
+
+class Overview extends React.PureComponent<Props> {
+
+  constructor (props: Props) {
+    super(props);
+  }
+
+  render () {
+    const accounts = keyring.getAccounts();
```

Comment:
This should be injected by the observable, so when things are added it is immediately available (which is important once the add is there) Just look for `allAccounts`, like it typically gets injected.